### PR TITLE
Two Pointers Algorithm added in PHP/Searches/TwoPointers.php

### PR DESCRIPTION
**Pull request by arafinmridul**
Definition, Usage and Step by step Two Pointers technique implementation in PHP with example problem. Explained in every step with proper comments.

### CODE REVIEW

File: Searches/TwoPointers.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            $ans++; # incrementing the $ans
+            // now we will move one of our pointers, otherwise it'll run forever
+            $left++; # doesn't matter which one we move
+        }
+        // The loop will go until the pointers point to the same element
+        // This will perfectly calculate the number of pairs
+    }
+    return $ans; # returning the number of pairs
+}
+echo TwoPointers($arr, $target); # outputs 3 as we've seen it
```
Comment (darwinz) [inline Searches/TwoPointers.php:77]: Please create a proper unit test, and remove this and the two inputs above

**Comment by arafinmridul** [comment]
For the ./Searches/TwoPointers.php file, Added file path in Directory.md. Added unit test cases.

**Comment by darwinz** [comment]
Looking much better.  Thank you for adding the file path in Directory.md, and adding a proper unit test.  The linter is complaining about one issue.  Fix this issue, and we should be able to merge.

```
FILE: /home/runner/work/PHP/PHP/Searches/TwoPointers.php
------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------
 1 | ERROR | [x] Header blocks must be separated by a single blank line
------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------
```

**Comment by darwinz** [comment]
Thanks for updating, @arafinmridul .  Looks like you still have a linter issue.  Please address this.

```
FILE: /home/runner/work/PHP/PHP/Searches/TwoPointers.php
------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------
 1 | ERROR | [x] Header blocks must be separated by a single blank line
------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------
```

If you run `vendor/bin/phpcs -n` locally after making a fix, you can see any errors and ensure there aren't any after each fix you do.

File: Searches/TwoPointers.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Two Pointers Approach
```
Comment (darwinz) [inline Searches/TwoPointers.php:3]:
```suggestion
<?php

/**
 * Two Pointers Approach
```

**Review by darwinz — APPROVED** [review]
Thank you!

**Pull request merged**